src/app/compressed-air-inventory/compressed-air-inventory-setup/compressed-air-catalog/compressed-air-catalog.module.ts

```diff
@@ -57,7 +59,9 @@ import { UnloadPointCatalogComponent } from './performance-points-catalog/unload
     OperatingHoursModalModule,
     ConfirmDeleteModalModule,
     ConnectedInventoryModule,
-    RouterModule
+    RouterModule,
+    NgbModule,
```

Comment:
Check some of your added and existing imports here. Some don't appear to be used.